server/.empirica/treatments.yaml

```diff
@@ -1,19 +1,40 @@
 factors:
```

Comment:
You may need to update your branch to the most recent commits to main. In talking with Nicolas, it turns out that there shouldn't be a .empricia directory under either server or client, so you'll see that is removed from the most recent commits in main.